opm/output/eclipse/EclipseIO.hpp

```diff
@@ -274,6 +274,19 @@ class EclipseIO
     /// \return Collection of per-cell, per-well, per-connection,
     /// per-segment, per-group, and per-aquifer dynamic results at
     /// simulation restart time.
+
+    void writeTimeStep(const Action::State&      action_state,
```

Comment:
You need to move this declaration away from this location.  In its current location, the declaration is between the documentation block for `loadRestart()` and the actual declaration for that function.  That will break the Doxygen-based documentation generation.

I would prefer moving this new function declaration [ahead of the function documentation](https://github.com/arturcastiel/opm-common/blob/30a642dbc889bc24c4c15982849ad99f1d40d1f7/opm/output/eclipse/EclipseIO.hpp#L231) for `loadRestart()` and, preferably, giving it a Doxygen-style documentation block of its own.